CI/circle_parallel.sh

```diff
@@ -12,6 +12,7 @@ if [ "$NODE_INDEX" = "1" ]; then
   #cp CI/pom.xml.circleci pom.xml
   java -version
   mvn --quiet verify -Psamples.circleci
+  mvn --quiet javadoc:javadoc -Psamples.circleci
```

Comment:
Discussed with @wing328 on Gitter, this is a way to check that Javadoc can be built